### ISSUE TITLE
QrcClient is not a constructor

### ISSUE THREAD

**Issue by patrickgilsf** [qsys-tools/qrc-client-js#1]
Is there user error here? I am trying to import and use this tool, and getting QrcClient is not a constructor. 

Thanks

**Comment by joshuaburkhalter** [comment]
> Is there user error here? I am trying to import and use this tool, and getting QrcClient is not a constructor.

There is a small problem with the way that QrcClient is getting exported.  You can get around it by doing:

const client = new QrcClient.default();

**Comment by patrickgilsf** [comment]
Thanks @joshuaburkhalter - I am actually making my own JS wrapper for this API as well. Should have a repo up in the next few weeks.

But I appreciate the info!

**Comment by joshuaburkhalter** [comment]
That's awesome @patrickgilsf , I'd love to help - are you implementing everything that QRC can do (mixer, pa routing, etc.)?  I've been wanting to build a full module that includes the QRC and PARAPI.

**Comment by patrickgilsf** [comment]
No, I am not. Just the calls that fit my use cases. Component Get, Component Set, Component GetComponents, Component GetControls.

Mixer, PA routing, Snapshots, can all be controlled by the above methods. 

The idea is to have a fully agnostic Q-Sys control, so you can control from a webpage, script, etc.....don't need to rely on a touch panel

Last, there is another one out there has the same goal - https://github.com/locimation/js-qrc

**Comment by jamestalmage** [comment]
Fixed in release `0.2.0`
